src/components/projects-summary/projects-summary.controller.ts

```diff
@@ -85,6 +86,10 @@ export class ProjectsSummaryController implements angular.IController {
         nextResource.href = this.Constants.HELP_BASE_URL + this.Constants.HELP[nextResource.help];
       }
     });
+
+    this.$scope.$watch(this.getgetRecentlyViewedItemsFromStorage, (newValue: any) => {
```

Comment:
I'm not sure what the overhead of `localStorage.getItem` is, but `$watch` is evaulated a lot. Can we just update the recent items on init and when the user clicks on one? We could use `$rootScope.$emit` when the services view updates the list and `$rootScope.$on` to handle it here.